redisinsight/ui/src/pages/home/components/add-database-screen/components/connectivity-options/styles.module.scss

```diff
@@ -2,29 +2,72 @@
   font-weight: 400 !important;
   display: flex;
   align-items: center;
+}
 
-  .cloudIcon {
-    margin-right: 8px;
+.cloudSection {
+  .typeBtn {
+    min-width: 160px !important;
   }
 }
 
 .typeBtn {
+  position: relative;
   width: 100%;
-  height: 76px !important;
+  height: 84px !important;
+  padding: 0 12px;
 
   border-color: var(--separatorColorLight) !important;
   color: var(--buttonSecondaryTextColor) !important;
   box-shadow: none !important;
 
-  &.primary {
```

Comment:
Looks like we still have a component that uses `.primary`. Either we should remove this style from the component, or keep it here

Reply:
yes, removed